README.md

```diff
@@ -1,10 +1,113 @@
+
+Negative click types can be used to mark where the object is __not__ located. Typically, these should only be used when correcting SAM2 mask outputs, because they can confuse SAM2 and impair performance if used too much. 
+
+Bite clicks can be used to mark the location of bites, or any other discrete behavioral event. 
+
+### Fish Family
+
+This button can be toggled to specify which fish family, or object category, you are marking annotations for. The current options are Parrotfish, Surgeonfish, Damselfish, and Other.
+
+### Fish Name
+
+This option specifies which individual you are marking annotations for. This should be a unique integer for every individual. 
+
+### Adding Annotations
+
+When the SAM2 Start frame, click type, fish family, and fish name are set to the correct values, use your cursor to click on the video where you wish to mark a click. Then press Enter or the Add Annotation button to record the annotation in the annotation table in the right panel. The Annotation Table will display the Frame that was annotated, the Click Type used (0 is a negative click, 1 is a positive click, 2 is a bite click), the fish name, family and the x y coordinates of the click location.
+
+> [!Note] 
+> Only frames that will be extracted for SAM2 processing should be annotated with positive / negative clicks or entries/exits. These frames will display a message "SAM2 Frame: Annotate Fish Position". 
+> Bite clicks can be annotated on any frame. 
+
+Annotations can be deleted using the Delete Selected or Delete All buttons. The Delete All button will prompt for confirmation before deleting all annotations.
+
+### Entries and Exits
+
+On the first SAM2 frame where the individual is visible, an Entry should be marked using the Add Entry button or the keystroke 'e'. On the last SAM2 frame where the individual is visible, an Exit should be marked using the Add Exit button or the keystroke 'x'. No location is associated with entries and exits, but the correct fish family and individual name should be specified before marking entries and exits. 
+Multiple entries and exits can be associated for each object; these will essentially tell SAM2 when to start and stop tracking the object. If the object leaves the field of view temporarily, it is best to mark it as an exit and re-entry; otherwise, SAM2 will keep looking for that object and may incorrectly mask a different but similar-looking object.
+
+If your cursor is in one of the input text boxes, pressing "e" or "x" will simply type in the text box and not create a new entry or exit. Click on the annotation table to remove your cursor from the input text box and use the hotkey shortcuts.
+
+### Saving Annotations
+
+To save your annotations, write the file name you wish to save as in the provided input textbox. If this file name has already been used, it will overwrite any existing files. 
```

Comment:
```suggestion
To save your annotations, write the file name you wish to save it as in the provided input textbox. If this file name has already been used, it will overwrite any existing files. 
```